gradle.properties

```diff
@@ -3,6 +3,7 @@ dokkaPluginVersion = 0.9.17
 kotlinPluginVersion = 1.3.10
 versionsPluginVersion = 0.20.0
 
+antlrVersion = 4.7.1
```

Comment:
Would it make sense to have a separate commit for integrating the library & plugin ?

Reply:
Probably, but I think in the past we usually introduced a dependency in the same commit as we started using it. It's only a few lines anyway, I think it makes more sense if the integration itself takes actual work beyond a very basic configuration.

Reply:
yes makes sense, actually forgot the nit at the beginning - maybe not even that.